src/lib/server/xata.ts

```diff
@@ -0,0 +1,203 @@
+// Generated by Xata Codegen 0.22.3. Please do not edit.
```

Comment:
@getneil Should this file actually be named xata.codegen.server.ts to indicate that it was auto-generated?  And then xata.ts will contain the code that passes the API key to the xata client instantiation (same as xata.js does now).

Reply:
not sure can we specify which file for them to generate?

that is why never I edited or rename it.

Reply:
At the moment, the file that is called xata.ts is what used to be called xata.codegen.server.js.  And the file called xata-old.ts is what used to xata.js.

It might be possible to change how the xata codegen tool names it, but maybe it's simpler to just use xata.codegen.server.ts for this file and rename xata-old.ts to xata.ts?

Reply:
it can be edited by .xatarc. will update this later.

Reply:
this should be done now